Templates/CSharp/EdgeDriverTemplate/ProjectTemplates/CSharp/Test/EdgeDriverTemplate.vstemplate

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<VSTemplate Version="3.0.0" xmlns="http://schemas.microsoft.com/developer/vstemplate/2005" Type="Project">
+  <TemplateData>
+    <TemplateID>Microsoft.CSharp.UnitTestProject.EdgeDriverTest</TemplateID>
+    <Name>Web Driver Test for Edge (.NET Framework)</Name>
+    <Description>Create an automatic UI test using Edge Driver.</Description>
+    <Icon Package="{52CBD135-1F97-2580-011F-C7CD052E44DE}" ID="531"/>
+    <ProjectType>CSharp</ProjectType>
+    <SortOrder>100</SortOrder>
+    <CreateNewFolder>true</CreateNewFolder>
+    <DefaultName>EdgeDriverTest</DefaultName>
+    <ProvideDefaultName>true</ProvideDefaultName>
+    <CreateInPlace>false</CreateInPlace>
+    <RequiredFrameworkVersion>4.6</RequiredFrameworkVersion>
```

Comment:
Is the framework version requirement real? Is 46 the lowest version supported by selenium or can other (potentially lower) versions also work?

Reply:
Seems like 4.5 is minimum because of MS Test